apps/prairielearn/src/server.ts

```diff
@@ -1736,6 +1736,18 @@ export async function initExpress(): Promise<Express> {
       coreElements: false,
     }),
   );
+  app.use(
+    '/pl/public/course_instance/:course_instance_id(\\d+)/assessment/:assessment_id(\\d+)/questions',
+    [
+      function (req, res, next) {
+        res.locals.navPage = 'public_assessment';
+        res.locals.navSubPage = 'questions';
+        res.locals.navbarType = 'public';
```

Comment:
Let's avoid this in favor of specifying these inline in `Navbar` in the page itself.

Reply:
done in 3a35b9749e6b83547eb14988f73709fa2d9075a5